content/articles/getting-started-clickfunnels.markdown

```diff
@@ -0,0 +1,222 @@
+---
+title: Getting Started With Your DNSimple Account After Transferring From ClickFunnels
+excerpt: A collection of articles to help you get started with your new DNSimple account quickly.
+categories:
+- DNSimple
+---
+
+# Getting Started With Your DNSimple Account After Transferring From ClickFunnels
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Welcome to [DNSimple](/articles/dnsimple-services/)! We are going to make your transition from ClickFunnels as simple as possible. [Simple](/articles/how-is-dnsimple-pronounced/) is in our name, and we're not going to let you down.
+
+## Step 1: Setting up a DNSimple account and trial
+
+<div class="section-steps" markdown="1">
+1. Go to [DNSimple.com](https://dnsimple.com). In the upper right corner of the screen, click the button that says <label>Get Started</label>.
+1. Complete the form using the **same email address that is associated with your ClickFunnels account**. Enter a password and confirm it. Click <label>Join DNSimple</label>.
+1. Now choose a plan. Select <label>Continue</label> under <label>Individual</label>, then choose <label>Get Started</label> under <label>Personal</label> or <label>Professional</label>, depending on your needs.
+
+    <tip>
+    [Learn the differences](/articles/personal-vs-pro/) between the Personal plan and the Professional plan.
+    </tip>
+
+1. Add your credit card details then click <label>Start my free trial</label>.
+
+    <info>
+    You can cancel at any time during your 30 day trial without being charged. If you choose to purchase products such as another domain or an SSL certificate during the free trial, your card will be charged. Read more about [account activation](/articles/account-activation/).
+    </info>
+</div>
+
+
+## Step 2: Asking ClickFunnels to move the domain to your DNSimple account
+
+<div class="section-steps" markdown="1">
+1. Send an email to [billing@clickfunnels.com](mailto:billing@clickfunnels.com) using the email address associated with your ClickFunnels account. You can use the below template, replacing the placeholders with your information.
+
+    > Hi there,
+
+    > I would like to move my domain(s) from ClickFunnels to my own DNSimple account.
+
+    > i. Confirm that you have made a paid DNSimple account
+
+    > YES
+
+    > ii. Provide the email address associated with your ClickFunnels and DNSimple accounts
+
+    > `your email address`
+
+    > iii. Name of domain(s) you wish to transfer
+
+    > `your domain(s)`
+
+    > iv. Verify the last 4 digits of the card we have on file for your ClickFunnels account
+
+    > `last 4 digits of your credit card`
+
+    > Thanks!
+
+1. ClickFunnels will reply to your email once they have moved the domain into your account. You will also receive an email from us at DNSimple letting you know the domain is waiting for you.
```

Comment:
Should we include a screenshot of the email we send?

Reply:
@SarahCaminiti1 good idea. can you please forward me the email? i know it's in the video, but i'd rather take a fresh screenshot and alter the text a bit (e.g., make it read example.com etc.).

Reply:
Sent

Reply:
hmm, I haven't seen this come thru yet.